contents/blog/posthog-cloud-eu.md

```diff
@@ -0,0 +1,53 @@
+date: 2022-10-10
+title: Introducing PostHog Cloud EU
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Product updates”]
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+--
+
+Today, we’re delighted to announce the [launch of PostHog Cloud EU](/eu) – a fully-managed version of PostHog where all data is hosted exclusively in the EU.
+
+Until now, running PostHog with GDPR compliance meant self-hosting on your own infrastructure, or comprehensively anonymizing data if using PostHog Cloud.
+
+Thanks to our new hosting option, customers in the EU, and any business that needs to keep user data within the EU, can get up and running on PostHog Cloud EU in just a few minutes.
+
+<GDPRForm />
+
+## FAQ
+
+### Where are PostHog Cloud EU's servers based?
+
+PostHog Cloud EU is hosted on AWS servers based in Frankfurt, Germany.
+
+### What data is transferred from the EU to the US?
+
+None. PostHog Cloud EU is an entirely independent instance of PostHog. All event data, user data, and the product itself, is hosted on our EU-based servers.
+
+### How do I use PostHog Cloud EU in compliance with GDPR?
+
+Just deploying PostHog Cloud EU isn't enough – you still need to [acquire consent](/tutorials/react-cookie-banner) from your users and comply with all other provisions of the GDPR, such as the right to be forgotten. Please see our [GDPR guidance](/docs/privacy/gdpr-compliance) for more information.
+
+### I'm an existing PostHog customer, can I migrate to PostHog Cloud EU?
+
+Yes. Please read our guide to [migrating to PostHog Cloud EU](/tutorials/migrate-eu-cloud). Please note: at this time, only event and person data can be migrated. Insights, dashboards, and feature flags are not migratable.
```

Comment:
@ellie I believe the above is all correct, but please verify I'm not telling fibs.

Reply:
No fibs here! 🙌